### ISSUE TITLE
Customer Facing Error Page (500 error, etc.)

### ISSUE THREAD

**Issue by randyabramson** [18F/crt-portal#707]
**As a product owner, I want users of civilrights.justice.gov to see a welcoming and directional error page when they come to our site and a script error occurs**

### Acceptance Criteria

Given that I am on the civilrights.justice.gov domain
When I hit a page and a server error or other error occurs
Then I see an error page page

Given that I am on the civilrights.justice.gov domain
When I hit a page and a server error or other error occurs
AND I click the English/Spanish toggle at the bottom of the page
Then I see the error page in Spanish

### Background Research

See the attached for a group of government focused 404 pages that we can also use as guidance for an error page. I included the features that I liked below most. Note that there are some very BAD 404s out there as well....namely: https://www.fbi.gov/r

[https://app.zenhub.com/files/183258764/23c5e9f9-b3d0-4358-b7b3-64eba05a5361/download](https://app.zenhub.com/files/183258764/23c5e9f9-b3d0-4358-b7b3-64eba05a5361/download)

### Page Elements

**Mandatory Text**
 We're sorry - something went wrong. {include error number here}
 We hope to fix this issue shortly. 

If you need to contact us immediately, you can reach us by:

**Email** (mailto: ask.crt@usdoj.gov ) (please use mail icon in footer)

**Phone (please use phone icon in footer):**

(202) 514-3847
1-855-856-1247 (toll-free)

Telephone Device for the Deaf
(TTY) (202) 514-0716"

**Mail (please use mail icon in footer):**

U.S. Department of Justice
Civil Rights Division
950 Pennsylvania Avenue, NW
Washington, D.C. 20530-0001

Other information: 
* <a href = "https://civilrights.justice.gov/#about-the-division">About the Civil Rights Division</a>
* If you or someone else is in immediate danger, <a href = "tel:911">please call 911 or local police.</a>

### **Other Assumptions**

* This error page will have the same look and feel as the rest of the site.
* We're assuming since there are only 3 pages on the site that if there is a script error happening that the form is down. If there are scenarios where the form and errors can exist in parallel, then we should consider adding this text to the 'reach us by' area: "If you need to contact us immediately, you can <a href = 'https://civilrights.justice.gov/report/">submit a report</a> or reach us by:"
* There are a ton of errors beyond 500 (or 404 for that matter). It is assumed that this page will display the correct error number as it is relevant.  A list of common errors are here: https://blog.pixelfish.com.au/most-common-website-errors

**Nice to haves**

1. Lead art: I would love a welcoming image, but I understand that it might be hard to get everyone to agree on what that is. I could see these as possibilities:
* A broken puzzle piece
* An unplugged plug
* A laptop with the words 'Error' on the screen

2. I'd consider image buttons for the following if we can come up with universally understood/approved images for:
- <a href = "https://civilrights.justice.gov/report/">File a complaint</a>
- Contact us by <a href = "https://civilrights.justice.gov/#contact-crt">phone or mail</a>
- <a href = "https://civilrights.justice.gov/#about-the-division">About the Civil Rights Division</a>
- If you or someone else is in immediate danger, <a href = "tel:911">please call 911 or local police.</a>

**Comment by randyabramson** [comment]
Jacklynn mock attached

![404-mock-jacklynn.png](https://images.zenhubusercontent.com/5ef156d51dd4f26e61414bbf/0cb20748-7fe5-442c-b8a4-52791c51daf1)

**Comment by randyabramson** [comment]
Reiterating assumption number three in case it was overlooked in the above: * There are a ton of errors beyond 500 (or 404 for that matter). It is assumed that this page will display the correct error number as it is relevant.  A list of common errors are here: https://blog.pixelfish.com.au/most-common-website-errors

**Comment by randyabramson** [comment]
Translation here[https://app.zenhub.com/files/183258764/40aa60fb-b060-44ff-b165-3c537f3113e7/download](https://app.zenhub.com/files/183258764/40aa60fb-b060-44ff-b165-3c537f3113e7/download)

**Comment by hakhalid11** [comment]
@randyabramson quick clarification, I am assuming this card if for all other error codes (500, 503, etc.) except for error 404.

**Comment by randyabramson** [comment]
@hakhalid11 this is in the assumptions section. There is a good list of errors here:  https://blog.pixelfish.com.au/most-common-website-errors

**Comment by hakhalid11** [comment]
@randyabramson @blaughman I got some good comments from @cantsin about this cards PR review. He made some good points about using our existing message and help text for our errors (ex. 500, 503, 403). In the initial PR I made the error page look like the design mock posted above which did not contain the extra helpful information so I removed it, but I think it would be beneficial to use these messages to provide extra information about these errors. I posted a screenshot of an updated design with some example error codes and their different messages. Also note that these messages already have Spanish translation which is a big plus.

### **Error 500**
![image.png](https://images.zenhubusercontent.com/5efb8164bf9ea016ca6016f2/65c30ee1-c2d5-4247-88dd-d58288419600)
### **Error 403**
![image.png](https://images.zenhubusercontent.com/5efb8164bf9ea016ca6016f2/52afcec2-f8bd-4e91-bcb5-d59a0cb0ff7c)
I welcome any feedback on this, I think this is a quick change regardless if we keep the extra text or not.

**Comment by hakhalid11** [comment]
@randyabramson let me know if the below works
I kept the "we hope to fix this shortly" using the original screenshot

![image.png](https://images.zenhubusercontent.com/5efb8164bf9ea016ca6016f2/d5607bb5-ec0d-45b5-bf48-aa560bec3133)

**Comment by randyabramson** [comment]
That works, @hakhalid11 !

**Comment by randyabramson** [comment]
@hakhalid11  how do I test for this?

cc: @blaughman

**Comment by hakhalid11** [comment]
@randyabramson great question I don't see a direct way to invoke error codes we are using with this card. I'd like to loop in @Jkrzy to see if he has any thoughts on this.
